source/solvers/navier_stokes_base.cc

```diff
@@ -2483,6 +2483,102 @@ NavierStokesBase<dim, VectorType, DofsType>::
     }
 }
 
+template <int dim, typename VectorType, typename DofsType>
+void
+NavierStokesBase<dim, VectorType, DofsType>::output_newton_update_norms(
+  const unsigned int display_precision)
+{
+  if constexpr (std::is_same_v<VectorType, GlobalVectorType> ||
+                std::is_same_v<VectorType,
+                               LinearAlgebra::distributed::Vector<double>>)
+    {
+      FEValuesExtractors::Vector velocities(0);
+      FEValuesExtractors::Scalar pressure(dim);
+
+      ComponentMask velocity_mask = fe->component_mask(velocities);
+      ComponentMask pressure_mask = fe->component_mask(pressure);
+
+      const std::vector<IndexSet> index_set_velocity =
+        DoFTools::locally_owned_dofs_per_component(dof_handler, velocity_mask);
+      const std::vector<IndexSet> index_set_pressure =
+        DoFTools::locally_owned_dofs_per_component(dof_handler, pressure_mask);
+
+      double local_sum = 0.0;
+      double local_max = DBL_MIN;
```

Comment:
```suggestion
      double local_max = -std::numeric_limits<double>::max();
```
is a better way to write this. The max you want it to start at -max, this way 0 can still be a max :)

Reply:
One can even use std::numeric_limits<double>::lowest() to avoid the -max notation